package.json

```diff
@@ -46,6 +46,7 @@
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "eslint-loader": "^1.9.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
```

Comment:
This is a peer dependency of `eslint-config-airbnb` and will be installed automatically. What was the motivation behind explicitly adding it here? Any advantages?

Reply:
I got an error initially, `ERROR in ./src/index.jsx
Module build failed: Error: Failed to load plugin jsx-a11y: Cannot find module 'eslint-plugin-jsx-a11y'
Referenced from: /Users/andeladeveloper/Desktop/andela-societies-frontend/node_modules/eslint-config-airbnb/index.js`

Reply:
the configuration for  `jsx-a11y/anchor-is-valid` requires this